src/main/java/net/ltxprogrammer/changed/command/CommandTransfur.java

```diff
@@ -145,23 +147,31 @@ private static int transfurPlayer(CommandSourceStack source, ServerPlayer player
             throw NOT_CAUSE.create();
         }
 
+        CheckCondition compatibility = new CheckCondition(player);
+        compatibility.checkOriginCondition(source.getServer());
```

Comment:
`checkOriginCondition()` is now called twice here, once in the `CheckCondition` constructor, and another in the command function.